config/locales/devise.en.yml

```diff
@@ -42,6 +43,7 @@ en:
       signed_up_but_inactive: "You have signed up successfully. However, we could not sign you in because your account is not yet activated."
       signed_up_but_locked: "You have signed up successfully. However, we could not sign you in because your account is locked."
       signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please follow the link to activate your account."
+      signed_up_but_not_approved: 'You have signed up successfully but your account has not been approved by your administrator yet'
```

Comment:
This message should follow the structure of the other messages for consistency sake. You should end with a period, and there should be a comma before but. Double quotes would be nice as well in case someone changes the message to have embedded single quotes. It would be great if you could also fix not_approved and updated_but_not_signed_in in order to match this standard.

Reply:
I'm not exactly sure what changes you want, but I changed them